Week 2/shopee/src/components/Cart.jsx

```diff
@@ -46,25 +54,12 @@ const Cart = () => {
                     min={1}
                     type="number"
                     value={product.quantity}
-                    onChange={(e) =>
-                      dispatch({
-                        type: "CHANGE_QUANTITY",
-                        payload: {
-                          id: product.id,
-                          quantity: e.target.value,
-                        },
-                      })
-                    }
+                    onChange={(e) => handleChangeQuantity(e, product)}
```

Comment:
@phamhuutinh0605 : Không được truyền arrow function trực tiêp trong props vì nó ảnh hưởng tới performance e sửa lại cái handleChangeQuantity để sử dụng dc như suggestion của a bên dưới nhé, check lại mấy chổ khác rồi sửa luôn
```suggestion
                    onChange={handleChangeQuantity(product)}
```